sections/03_categorize.md

```diff
@@ -645,3 +679,4 @@ LSTMs) [@arxiv:1602.00357]. This may be a particular area of opportunity for
 deep neural networks. The ability to recognize relevant sequences of events from
 a large number of trajectories requires powerful and flexible feature
 construction methods -- an area in which deep neural networks excel.
+>>>>>>> greenelab/master
```

Comment:
Remove this line

Reply:
OK

Reply:
Trying to fix the error.

Reply:
I have corrected all places according to reviews of @dhimmel.  
(a bit confused by how to update my pull requests when my fork is quite behind the main. not sure if I have done correctly!)